### PR TITLE
Semvecpy

### PR DESCRIPTION
**Pull request by hannahburkhardt**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by jusger** [comment]
Just tagging in. It looks to me like are recipes might be correct but our pypi package isn't installing correctly. I'll look into things a bit and see if I can't track down the issue.

File: recipes/semvecpy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 10c62aab3a8bb387f1f9b132b74ea266d2331c0a05fce5b6f911b2972e08dfd9
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py2k]
```
Comment (marcelotrevisani) [inline recipes/semvecpy/meta.yaml]: Could you please change it to be ``noarch``?
You can select python on host and run

```suggestion
  noarch: python
```

File: recipes/semvecpy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+requirements:
+  host:
+    - pip
+    - python
```
Comment (marcelotrevisani) [inline recipes/semvecpy/meta.yaml]:
```suggestion
    - python >3.5
```

File: recipes/semvecpy/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  run:
+    - python
```
Comment (marcelotrevisani) [inline recipes/semvecpy/meta.yaml]:
```suggestion
    - python >3.5
```

File: recipes/semvecpy/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    - semvecpy.vectors
+
+about:
+  home: https://github.com/semanticvector/semvecpy
+  license: BSD
+  license_family: BSD
+  license_file: LICENSE.txt
+  summary: Semantic Vectors work in Python
+  doc_url: 
+  dev_url:
```
Comment (marcelotrevisani) [inline recipes/semvecpy/meta.yaml]: Could you please delete those empty attributes? ``doc_url`` and ``dev_url``

**Pull request merged**